Microsoft.VisualStudio.Terminal.Embeddable/ITerminal.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.Terminal
+{
+    [ComImport, Guid("E195D61C-2821-49F1-BE0E-B2CD82F1F856")]
+    public interface ITerminal
+    {
+        Task ShowAsync();
```

Comment:
Up to you, but you may want to comment these guys so they show up when consumers of the API try to use it.